src/azure-cli-core/azure/cli/core/_help.py

```diff
@@ -192,8 +195,8 @@ def update_loaders_with_help_file_contents(self, nouns):
 
     # This method is meant to be a hook that can be overridden by an extension or module.
     @staticmethod
-    def example_provider(help_file):
-        return help_file.examples
+    def check_for_new_examples(command):  # pylint: disable=unused-argument
+        return []
```

Comment:
We can use the name `check_for_ai_examples` to make the purpose clearer.

Reply:
That makes sense. How about `update_examples`?  `check_for_new_examples` is also fine to me. It's up to you. 😊

Reply:
`update_examples` sounds good to me!